buildscripts/gateway-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+export SERVER_ENDPOINT=127.0.0.1:24240
```

Comment:
Any reason to pick a high port?

Reply:
avoiding port conflicts when local testing without docker